Calmcoder.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+  <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These two spaces you added here made the XML document invalid and caused it to not be detected by Gedit.